### ISSUE TITLE
Add Karify

### ISSUE THREAD

**Issue by mennozuidema** [2factorauth/twofactorauth#5087]
- [x] I have searched both open and closed [issues](https://github.com/2factorauth/twofactorauth/issues). The issue I'm creating is not a duplicate of an existing issue.
- [x] I have searched both open and closed [pull requests](https://github.com/2factorauth/twofactorauth/pulls). The issue I'm creating is not a duplicate of an existing pull request.

### Information about the site to be added that supports 2FA: ###

* Site name - `Karify`

* Site URL - `karify.com`

* Does the site support 2FA? - `Yes`

* Documentation URL - `https://support.karify.com/en/support/solutions/articles/4000147820`

### Information about what methods of 2FA the site supports: ###

* Software:
  - [x] TOTP (RFC-6238, Google Authenticator)
  - [ ] Proprietary (Authy, Duo, etc.)

* Hardware:
  - [ ] FIDO2/U2F security keys
  - [ ] Other

- [ ] Phone calls
- [x] SMS tokens
- [ ] Email tokens

### Information about the site's eligibility to be added to twofactorauth.org: ###

- [x] I have checked that the site meets twofactorauth.org's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md).
- [ ] The site is ranked within the top 200,000 sites on [Alexa](https://www.alexa.com/siteinfo/) or has a substantial social media following.
- [x] The site does not contain or promote content that violates twofactorauth.org's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md), including pornographic, discriminatory, or unlawful content.

**Comment by Carlgo11** [comment]
Unfortunately, as the site is ranked above 200K on [Alexa](https://www.alexa.com/siteinfo) it doesn't meet our guidelines of [new site additions](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#guidelines).

Sorry for the inconvenience.

**Comment by mennozuidema** [comment]
Thanks @Carlgo11 for checking this issue. Since some password managers like 1password are using twofactorauth.org to empathise on using this and making their accounts more secure, I would like to add this site and help users make the right decision. What is the reason to limit on the top 200K on Alexa, and will this ever change?

**Comment by Carlgo11** [comment]
We've decided to put a limit on new additions to only the 200 000 most popular sites as it takes work to maintain the lists.
200 000, we've found to be a good middle ground between maintainability and usability.

What other companies use our data for is beyond our reach. You might be able to have the people at 1Password add it to their own dataset? I'm not sure how their setup works as the integration of our data is something they've done by themselves.
